js/search/Autocomplete.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useRef, useState } from "react";
+import { render } from "react-dom";
+
+import { useSearchBox } from "react-instantsearch";
+import { autocomplete } from "@algolia/autocomplete-js";
+import { Fragment } from "react";
```

Comment:
I think the imports can be combined into one line: 

```suggestion
import React,{ useEffect, useRef, useState, render, Fragment } from "react";
import { useSearchBox } from "react-instantsearch";
import { autocomplete } from "@algolia/autocomplete-js";
```